src/App.vue

```diff
@@ -1,30 +1,27 @@
 <template>
-  <div id="nav">
-    <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
-  </div>
-  <router-view/>
+  <img alt="Vue logo" src="./assets/logo.png" />
+  <HelloWorld msg="Hello Vue 3 + TypeScript + Vite" />
 </template>
 
+<script lang="ts">
+import { defineComponent } from 'vue'
+import HelloWorld from './components/HelloWorld.vue'
```

Comment:
Somthing more to think about is the bring back the resolvers like webpack has. IE the `@` for `/src` but we can make a seperate issue for that

Reply:
Yeah, I'll try and set an @-alias 👍